brickflow/engine/task.py

```diff
@@ -742,8 +742,17 @@ def databricks_task_type_str(self) -> str:
 
     @property
     def builtin_notebook_params(self) -> Dict[str, str]:
-        # 2 braces to escape for 1
-        return {i.value: f"{{{{{i.name}}}}}" for i in BrickflowBuiltInTaskVariables}
+        mapping = {
+            "brickflow_job_id": "{{job.id}}",
```

Comment:
Can we not just change the `BrickflowBuiltInTaskVariables` class to something like below and make changes accordingly? This will avoid maintaining the mapping at both places

```
class BrickflowBuiltInTaskVariables(Enum):
    # key is the {{ }} and value is the key
    brickflow_job_id = "{{job.id}}"
    brickflow_run_id = "{{task.run_id}}"
    ...
```

Reply:
@naresh483 do we need to close this pr?